node/silkworm/rpc/backend_server.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+   Copyright 2022 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include "backend_server.hpp"
+
+#include <silkworm/common/endian.hpp>
+#include <silkworm/common/log.hpp>
+#include <types/types.pb.h>
+
+namespace silkworm::rpc {
+
+inline types::H128* new_H128_from_bytes(const uint8_t* bytes) {
+    auto h128{new types::H128()};
+    h128->set_hi(endian::load_big_u64(bytes));
+    h128->set_lo(endian::load_big_u64(bytes + 8));
+    return h128;
+}
+
+inline types::H160* new_H160_address(const evmc::address& address) {
```

Comment:
Better add `static` for [internal linkage](https://google.github.io/styleguide/cppguide.html#Internal_Linkage).